### PR TITLE
[PFCWD]: Add support for t0 toplogy and arista fanout testbed

### PR DESCRIPTION
**Pull request by sihuihan88**
Signed-off-by: Sihui Han <sihan@microsoft.com>

### Description of PR

Fixes # (issue)

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Approach
How did you do it?
Add support for t0 toplogy and arista fanout testbed
How did you verify/test it?
Tested on DUT
Any platform specific information?
Supported testbed topology if it's a new test case?

### CODE REVIEW

**Comment by lguohan** [comment]
@marian-pritsak , can you review?

File: ansible/roles/test/files/helpers/pfctest.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
```
Comment (marian-pritsak) [inline ansible/roles/test/files/helpers/pfctest.py]: Script was taken from another repo. Can you please check if licenses are compatible?

File: ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml [hunk 2]
```diff
@@ -25,13 +27,30 @@
...
+
 - name: Set timers
   set_fact:
       pfc_wd_detect_time: 200
       pfc_wd_restore_time: 200
       pfc_wd_restore_time_large: 30000
       pfc_wd_poll_time: 100
 
+- name: Create pfc generator script in case it doesn't exist
+  file: path=/mnt/flash/pfctest.py state=touch
```
Comment (marian-pritsak) [inline ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml]: why touch?

Comment (sihuihan88) [reply ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml]: Copy the script will fail if the file doesn't exist on this platform. So create the file first.

File: ansible/roles/test/files/helpers/pfctest.py (hunk 1 above)
Comment (sihuihan88) [reply ansible/roles/test/files/helpers/pfctest.py]: Sure. I will make sure licenses before checking in.

File: ansible/roles/test/files/helpers/pfc_gen.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    parser.add_option("-r", "--rsyslog-server", type="string", dest="rsyslog_server", default="127.0.0.1", help="Rsyslog server IPv4 address",metavar="IPAddress") 
+    (options, args) = parser.parse_args()
+
+    if options.interface is None:
+        print "Need to specify the interface to send PFC packets."
+        parser.print_help()
+        sys.exit(1)
+
+    if options.time > 65535 or options.time < 0:
+        print "Quanta is not valid. Need to be in range 0-655535."
```
Comment (yxieca) [inline ansible/roles/test/files/helpers/pfc_gen.py]: Typo here? s/655535/65535 ?

**Pull request merged**